docusaurus/docs/metadata.md

```diff
@@ -63,12 +63,12 @@ Plugin dependency. Used to display information about plugin dependencies in the
 
 #### Properties
 
-| Property  | Type   | Required | Description                                        |
-| --------- | ------ | -------- | -------------------------------------------------- |
-| `id`      | string | **Yes**  |                                                    |
-| `name`    | string | **Yes**  |                                                    |
-| `type`    | string | **Yes**  | Possible values are: `app`, `datasource`, `panel`. |
-| `version` | string | **Yes**  |                                                    |
+| Property  | Type   | Required | Description                                                                                                                                   |
+| --------- | ------ | -------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`      | string | **Yes**  |                                                                                                                                               |
+| `name`    | string | **Yes**  |                                                                                                                                               |
+| `type`    | string | **Yes**  | Possible values are: `app`, `datasource`, `panel`.                                                                                            |
+| `version` | string | **Yes**  | We suggest using [Semver](https://semver.org/) for your plugin versioning to alert users about potential changes, new features, or bug fixes. |
```

Comment:
nit: should we mention that a Semver range is best here?

Reply:
@jackw this is the plugin version, not the grafana version. Unless I misunderstand your suggestion?

Reply:
Ah ok. Sorry for the misunderstanding I'd assumed it was the Grafana version! 👍